<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActionCommand</class>
 <widget class="QWidget" name="ActionCommand">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>222</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ActionCommand</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="1">
    <widget class="QLabel" name="title">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Action Command</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QWSwitcher" name="toggle" native="true">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: rgb(80, 80, 80);
border:0;
border-radius:8;
color: rgb(255, 255, 255);
}
QPushButton::hover{
background-color: rgb(80, 80, 80,200);
}
QPushButton::pressed{
background-color: rgb(80, 80, 80,100);
}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>5</number>
      </property>
      <item row="2" column="4">
       <widget class="QPushButton" name="button_reset_vel">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Ubuntu</family>
          <pointsize>9</pointsize>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QtMaterialTextField" name="lineEdit_vel_y" native="true">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_37">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Input Position</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignHCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QtMaterialTextField" name="lineEdit_pos_x" native="true">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_40">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Key Velocity</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignHCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QPushButton" name="button_apply_vel">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Ubuntu</family>
          <pointsize>9</pointsize>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QtMaterialTextField" name="lineEdit_vel_x" native="true">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="6">
       <widget class="QWidget" name="widget_3" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>70</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="2" column="0">
          <widget class="QWidget" name="widget_4" native="true">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>70</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>70</height>
            </size>
           </property>
           <widget class="QPushButton" name="button_down">
            <property name="geometry">
             <rect>
              <x>50</x>
              <y>40</y>
              <width>45</width>
              <height>30</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>45</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>45</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>S</string>
            </property>
            <property name="icon">
             <iconset resource="../../RUI.qrc">
              <normaloff>:/svg/svg/arrow-down.svg</normaloff>:/svg/svg/arrow-down.svg</iconset>
            </property>
           </widget>
           <widget class="QPushButton" name="button_left">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>40</y>
              <width>45</width>
              <height>30</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>45</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>45</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>A</string>
            </property>
           </widget>
           <widget class="QPushButton" name="button_right">
            <property name="geometry">
             <rect>
              <x>100</x>
              <y>40</y>
              <width>45</width>
              <height>30</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>45</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>45</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>D</string>
            </property>
           </widget>
           <widget class="QPushButton" name="button_up">
            <property name="geometry">
             <rect>
              <x>50</x>
              <y>0</y>
              <width>45</width>
              <height>30</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>45</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>45</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>W</string>
            </property>
            <property name="icon">
             <iconset resource="../../RUI.qrc">
              <normaloff>:/svg/svg/arrow-up.svg</normaloff>:/svg/svg/arrow-up.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
           </widget>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QWidget" name="widget_2" native="true">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>70</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>70</height>
            </size>
           </property>
           <widget class="QPushButton" name="button_turn_left">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>40</y>
              <width>45</width>
              <height>30</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>45</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>45</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>J</string>
            </property>
           </widget>
           <widget class="QPushButton" name="button_turn_right">
            <property name="geometry">
             <rect>
              <x>100</x>
              <y>40</y>
              <width>45</width>
              <height>30</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>45</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>45</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>L</string>
            </property>
           </widget>
           <widget class="QPushButton" name="button_turn_up">
            <property name="geometry">
             <rect>
              <x>50</x>
              <y>0</y>
              <width>45</width>
              <height>30</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>45</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>45</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>I</string>
            </property>
            <property name="icon">
             <iconset resource="../../RUI.qrc">
              <normaloff>:/svg/svg/arrow-up.svg</normaloff>:/svg/svg/arrow-up.svg</iconset>
            </property>
           </widget>
           <widget class="QPushButton" name="button_turn_down">
            <property name="geometry">
             <rect>
              <x>50</x>
              <y>40</y>
              <width>45</width>
              <height>30</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>45</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>45</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>K</string>
            </property>
            <property name="icon">
             <iconset resource="../../RUI.qrc">
              <normaloff>:/svg/svg/arrow-down.svg</normaloff>:/svg/svg/arrow-down.svg</iconset>
            </property>
           </widget>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_13">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Keyboard
input speed</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="button_reset_pos">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Ubuntu</family>
          <pointsize>9</pointsize>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QtMaterialTextField" name="lineEdit_pos_y" native="true">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QtMaterialTextField" name="lineEdit_pos_z" native="true">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="6">
       <widget class="Line" name="line_4">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>2</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>2</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #283149;</string>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QtMaterialTextField" name="lineEdit_vel_z" native="true">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QPushButton" name="button_apply_pos">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Ubuntu</family>
          <pointsize>9</pointsize>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWSwitcher</class>
   <extends>QWidget</extends>
   <header>qwool/qwswitcher.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QtMaterialTextField</class>
   <extends>QWidget</extends>
   <header>qt_material_widgets/qtmaterialtextfield.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../RUI.qrc"/>
 </resources>
 <connections/>
</ui>
